core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -811,6 +812,12 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val SHUFFLE_IO_PLUGIN_CLASS =
+    ConfigBuilder("spark.shuffle.io.plugin.class")
```

Comment:
Could we add "sort" into this config name, since it is only applied to sort shuffle? e.g. use name like "spark.shuffle.sort.io.plugin.class", or something else?